lib/browser/api/app.js

```diff
@@ -45,6 +45,14 @@ if (process.platform === 'darwin') {
   }
 }
 
+if (process.platform === 'linux') {
+  app.launcher = {
```

Comment:
Would you mind adding some spec coverage for these new APIs?

You could add them after this line: https://github.com/electron/electron/blob/6081cba15d0bd88e6eed1329318d0b1070faf1e4/spec/api-app-spec.js#L291

Not sure if Unity is running on our build servers or not but at least we could cover that calling these methods does not crash the app and they return data of expected types.